src/etc/rc.bootup

```diff
@@ -227,6 +229,9 @@ if (!$debugging) {
 /* re-make hosts file after configuring interfaces */
 system_hosts_generate();
 
+/* add configured trusted CA certificates to /etc/ssl/certs */
+ca_trustedcerts_generate();
+
```

Comment:
And completely unnecessary.  It's not like people are restoring or swapping configs in prod every other time they reboot.  Also not friendly to write cycle sensitive media.